src/editing_tools/AdjustTool.vala

```diff
@@ -418,10 +470,11 @@ public class EditingTools.AdjustTool : EditingTool {
         virgin_histogram_pixbuf = histogram_pixbuf.copy ();
 
         DataCollection? owner = canvas.photo.get_membership ();
-        if (owner != null)
+        if (owner != null) {
             owner.items_altered.connect (on_photos_altered);
+        }
 
-        base.activate (canvas);
+        activate (canvas);
```

Comment:
Hmm isn't the `base.` here necessary? This should call the original activate method of the base class

Reply:
Thinking about it, yeah, probably, or the call would have been this.activate